frontend/src/app/modules/landing/landing-page/landing-page.component.sass

```diff
@@ -0,0 +1,470 @@
+@import "src/styles/colors"
+
+@media screen and (min-width: 2540px)
+  .all
+    button
+        h3
```

Comment:
I believe we don't need so many new styles

Reply:
maybe i need to split landing page in few component, to make it easier to read, but i don't think that i could decrease variable amount for easy